pkg/nsx/services/common/policy_tree.go

```diff
@@ -52,8 +52,10 @@ func getNSXResourcePath[T any](obj T) *string {
 		return v.Path
 	case *model.TlsCertificate:
 		return v.Path
+	case *model.SharedResource:
+		return v.Path
 	default:
-		log.Error(nil, "Unknown NSX resource type %v", v)
+		log.Error(nil, "Get NSX resource path", "Unknown NSX resource type %v", v)
```

Comment:
`"Unknown NSX resource type %v",`
in the third arg, can we have the format arg %v? I recalled it should be a constant string.

Reply:
In the leafWrapper function:
```
default:
		log.Error(nil, "Unknown NSX resource type", v)
```
we have one more place to fix.